lib/iris/tests/unit/analysis/scipy_interpolate/test__RegularGridInterpolator.py

```diff
@@ -0,0 +1,85 @@
+# (C) British Crown Copyright 2015, Met Office
+#
+# This file is part of Iris.
+#
+# Iris is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Iris is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Iris.  If not, see <http://www.gnu.org/licenses/>.
+"""Unit tests for the :func:`iris.analysis.interpolate.linear` function."""
+
+from __future__ import (absolute_import, division, print_function)
+
+# Import iris.tests first so that some things can be initialised before
+# importing anything else.
+import iris.tests as tests
+
+import mock
+
+import numpy as np
+import iris
+
+from iris.analysis._scipy_interpolate import _RegularGridInterpolator
+from scipy.sparse.csr import csr_matrix
+import iris.tests.stock as stock
+
+
+class Test(tests.IrisTest):
+    def setUp(self):
+        '''Load a source cube, then generate an interpolator instance and
+        target grid.'''
+        self.cube = stock.simple_2d()
+        x_points = self.cube.coord('bar').points
+        y_points = self.cube.coord('foo').points
+        self.interpolator = _RegularGridInterpolator([x_points, y_points],
+                                                     self.cube.data,
+                                                     method='linear',
+                                                     bounds_error=False,
+                                                     fill_value=None)
+        newx = x_points + 0.7
+        newy = y_points + 0.7
+        self.test_increment = 0.60666667
```

Comment:
Reproduced the magic number through 'manual' linear interpolation (my first attempt on Friday I'd mixed up the co-ordinate axes). Since the test data is purely linear, the same increment applies at every point in the set.